client/app/components/game-client.js

```diff
@@ -0,0 +1,62 @@
+import Ember from 'ember';
+import GameChannel from 'vilive/utils/game-channel';
+
+export default Ember.Component.extend({
+  init() {
+    this.set('messages', Ember.makeArray());
+    GameChannel.init(this.get('session-manager').currentToken());
+    GameChannel.on('event', payload => {
+      this.putMessage({
+        server: {
```

Comment:
I fully expect us to have to "render" the message in code some how. I would think we would want to wrap item names, player names, and NPC names with `<span>` and different classes to change the styling or add links for quick commands. These are all much later concerns though since we'll have to actually have a game working before we can experiment with message structure and rendering.